src/Infrastructure.Styles.Analyzer/RMSYN1102SeparateExitConditionsAnalyzerTools/IfStatementSyntaxWrapperStrategy.cs

```diff
@@ -0,0 +1,57 @@
+﻿// This file is part of the re-motion Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// re-motion Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Infrastructure.Styles.Analyzer.RMSYN1102SeparateExitConditionsAnalyzerTools
+{
+  public class IfStatementSyntaxWrapperStrategy : IConditionalStatementWrapperStrategy
+  {
+    public void RemoveElseClause (ref SyntaxNode node)
+    {
+      var ifStatement = (IfStatementSyntax)node;
+      node = ifStatement.WithElse(null);
+    }
+
+    public void WithCondition (ref SyntaxNode node, SyntaxNode condition)
+    {
+      var ifStatement = (IfStatementSyntax)node;
+      node = ifStatement.WithCondition((ExpressionSyntax)condition);
+    }
+
+    public void WithElse (ref SyntaxNode node, ElseClauseSyntax elseClause)
+    {
+      var ifStatement = (IfStatementSyntax)node;
+      node = ifStatement.WithElse(elseClause);
+    }
+
+    public void ReintroduceParentElseClause (ref SyntaxNode node, ElseClauseSyntax elseClause)
+    {
+      elseClause =
+        elseClause.WithElseKeyword(
```

Comment:
indent / whitespace is incorrect. should be double-spaced. Please check rest of code, too.

Reply:
Not sure how they are not double spaced, I think I'm missing something.

Reply:
Single indent is two whitespace.
Double indent is two indents.
-> it's always indents we're talking about :)